### PR TITLE
Profile icon

### PR DESCRIPTION
**Pull request by Jamil-Bailony**
- [x] add two components in navbar one for buttons and the other for user icon and settings

- [x] render the user photo

- [x] style elements and fix the vertical alignment of the navbar elements

image

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -8,6 +8,11 @@
     "buttons": {
       "signIn": "Sign In",
       "signUp": "Sign Up"
+    },
+    "userSettings": {
+      "userIcon":"User icon",
+      "profile":"Your profile",
```
Comment (louisrli) [inline public/locales/en/translation.json]: spaces

File: src/components/Navbar/SignInUpButtons/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div>
+      <div>
+        <button type="button" className="signInBtn">
+          <a href="/signin">{t("navbar.buttons.signIn")}</a>
+        </button>
+        <button type="button" className="signUpBtn">
+          <a href="/signup">{t("navbar.buttons.signUp")}</a>
+        </button>
+      </div>
```
Comment (louisrli) [inline src/components/Navbar/SignInUpButtons/index.jsx]: unnecessary div

**Review by louisrli — CHANGES_REQUESTED** [review]

**Comment by Jamil-Bailony** [comment]
done! If there is anything else, just let me know please @louisrli 
I am glad that I found just two comments :smile:

File: src/components/Navbar/UserSetting/index.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    width: 3rem !important;
+    border-radius: 50%;
+    border: 2px solid $warningColor;
+    cursor: pointer;
+
+    &:hover {
+      background-color: rgba($color: #000000, $alpha: 0.2);
+    }
+  }
+  .dropdown-toggle {
```
Comment (hiba-machfej) [inline src/components/Navbar/UserSetting/index.scss:18]: space

**Review by hiba-machfej — APPROVED** [review]

**Review by jennetturkkan — APPROVED** [review]

**Review by louisrli — APPROVED** [review]

**Comment by Jamil-Bailony** [comment]
Hey @Ammar-64 :)

**Pull request merged**